extensions/vscode/src/language-server/index.ts

```diff
@@ -0,0 +1,54 @@
+import { createConnection, createServer, createSimpleProject } from '@volar/language-server/node';
+import { create as createYamlService } from 'volar-service-yaml';
+import { SchemaPriority } from 'yaml-language-server';
+import { frontmatterPlugin } from './languagePlugin';
+import { readSchema } from './schema';
+
+const connection = createConnection();
+const server = createServer(connection);
+
+connection.listen();
+
+connection.onInitialize((params) => {
+  const yamlService = createYamlService({
+    getLanguageSettings(_context) {
+      const schema = readSchema();
+
+      return {
+        completion: true,
+        validate: true,
+        hover: true,
+        format: true,
+        yamlVersion: '1.2',
+        isKubernetes: false,
+        schemas: [
+          {
+            uri: 'https://tutorialkit.dev/reference/configuration/',
```

Comment:
So this is needed and HAS to be a valid URI otherwise the yaml extension try to load it from disk.

This also shows up under the documentation of every prop and is clickable:

<img src="https://github.com/user-attachments/assets/23b7ef55-f851-418a-adaa-45cc975e2ebe" width="300">

It opens in your browser and I choose a link that shows the reference.

Reply:
Nit, but do we need the trailing slash?